airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -203,6 +214,8 @@ def read_records(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iter
                 raise e
 
             self.logger.warning(error_msg)
+        except GitHubAPILimitException:
+            self.logger.warning("Limits for all provided tokens were reached, please try again later")
```

Comment:
nit: "Limits for all provided tokens are reached, please try again later", since we are seeing the limits now, not in the past.